### PR TITLE
Couchbase - Added Get Auto Failover Settings operation

### PR DESCRIPTION
**Pull request by mihaitusa**
Added Get Auto Failover Settings operation && bring the integration closer to this days && refactoring

Signed off by Tusa Mihai: tusa.mihai@gmail.com

### CODE REVIEW

**Comment by mihaitusa** [comment]
@tethryus - How am I suppose to fix Java 7 build if the code is wrote using functional style (Java 8) ?

**Comment by Bonczidai** [comment]
@tethryus shouldn't we remove the `  - openjdk7` from Travis build?

**Comment by mihaitusa** [comment]
Isn't enough if it will perform build task using only oraclejdk8?

File: cs-couchbase/src/main/java/io/cloudslang/content/couchbase/entities/builders/HttpClientInputsBuilder.java [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+public class HttpClientInputsBuilder {
+    private static final String[] VALID_HOSTNAME_VERIFIERS = new String[]{"allow_all", "browser_compatible", "strict"};
+    private static final String ALLOW_ALL = "allow_all";
+
+    private HttpClientInputsBuilder() {
+        // prevent instantiation
+    }
+
+    public static HttpClientInputs buildHttpClientInputs(String username, String password, String proxyHost, String proxyPort,
```
[the remark this answers is not in the document]
Comment (mihaitusa) [reply cs-couchbase/src/main/java/io/cloudslang/content/couchbase/entities/builders/HttpClientInputsBuilder.java]: Right as always... working on it already

**Comment by Bonczidai** [comment]
@mihaitusa @tethryus I think it should be enough with `jdk8`

**Comment by mihaitusa** [comment]
moved to cs-couchbase

**Pull request closed without merging**